test/StreetNameRegistry.Api.BackOffice.IntegrationTests/StreetNameRegistry.Api.BackOffice.IntegrationTests.csproj

```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\..\packages\Be.Vlaanderen.Basisregisters.Build.Pipeline\Content\Be.Vlaanderen.Basisregisters.Build.Pipeline.Settings.Test.props" />
+
+  <ItemGroup>
+    <Content Include="appsettings.json" CopyToOutputDirectory="Always" />
+    <Content Include="sqlserver.yml">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </Content>
```

Comment:
Same remark

Reply:
this should actually stay, because a library will be made (replacing ContainerHelper) to load that yml file